weave/trace_server/async_batch_processor.py

```diff
@@ -54,7 +56,10 @@ def _process_batches(self) -> None:
                 current_batch.append(self.queue.get())
 
             if current_batch:
-                self.processor_fn(current_batch)
+                try:
```

Comment:
This blind catch-all feels like a hammer. Do we always want to catch in every exception case? Maybe. Worth investigating